cmd/pulumi-test-language/interface.go

```diff
@@ -662,6 +662,17 @@ func (eng *languageTestServer) RunLanguageTest(
 
 		sdkName := fmt.Sprintf("%s-%s", pkg, version)
 		sdkTempDir := filepath.Join(token.TemporaryDirectory, "sdks", sdkName)
+		_, err = os.Stat(sdkTempDir)
+		if err == nil {
+			// If the directory already exists then we don't need to regenerate the SDK
+			sdkArtifact, err := languageClient.Pack(sdkTempDir, version, artifactsDir)
```

Comment:
I suppose ideally we wouldn't even re-pack it as it should be in the artifacts folder already, but this is probably fine for now.

Reply:
Yeah, I think that would be better ideally.  But the artifact name is calculated in the fake language host, and we'll need some extra logic to make sure we can match that here.  I'd rather leave that until we have language support for all languages, then it's maybe easier to see the pattern there, or add additional methods in each language host.